.env.example

```diff
@@ -62,4 +62,7 @@ SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
 
 # How large a window we use to calculate moving average of radar values
 RADAR_WINDOW_SIZE_SECONDS=15
-RADAR_WINDOW_SIZE_SECONDS_TEST=1
\ No newline at end of file
+RADAR_WINDOW_SIZE_SECONDS_TEST=1
+
+# Threshold for sending out a new alert when door sensor has low battery
+LOW_BATTERY_ALERT_TIMEOUT_THRESHOLD = 86400
```

Comment:
You'll also need to include the `_TEST` version of this variable in the `.travis.yaml` file so that it will have a value when Travis runs the tests.

Reply:
Does the env variable also need to be in `sensor-deployment.yaml`?

Reply:
Yes! Good catch.